### PR TITLE
KFSPTS-30712 Add SSC Processor Field to IWNT Document

### PR DESCRIPTION
**Pull request by ajd299**

### CODE REVIEW

File: src/main/resources/com/rsmart/kuali/kfs/cr/businessobject/datadictionary/CheckReconciliation.xml [hunk 1]
```diff
@@ -275,6 +281,8 @@ http://www.springframework.org/schema/beans http://www.springframework.org/schem
               p:attributeName="status" />
         <bean parent="FieldDefinition"
               p:attributeName="sourceCode" />
+        <bean parent="FieldDefinition"
+              p:attributeName="active" />
```
Comment (nkimble) [inline src/main/resources/com/rsmart/kuali/kfs/cr/businessobject/datadictionary/CheckReconciliation.xml:285]: Are these changes part of KFSPTS-30132? I don't think this file should be in this PR.

File: src/main/resources/com/rsmart/kuali/kfs/cr/document/datadictionary/CheckReconciliationMaintenanceDocument.xml [hunk 2]
```diff
@@ -67,6 +67,9 @@
               p:required="false" />
         <bean parent="MaintainableFieldDefinition"
               p:name="versionNumber" />
+        <bean parent="MaintainableFieldDefinition"
+              p:name="active"
+              p:defaultValue="true" />
```
Comment (nkimble) [inline src/main/resources/com/rsmart/kuali/kfs/cr/document/datadictionary/CheckReconciliationMaintenanceDocument.xml:72]: Are these changes part of KFSPTS-30132? I don't think this file should be in this PR.

File: src/main/resources/com/rsmart/kuali/kfs/cr/ojb-cr.xml [hunk 3]
```diff
@@ -33,6 +33,7 @@
     	<field-descriptor name="bankCode" column="BNK_CD" jdbc-type="VARCHAR" />
     	<field-descriptor name="cancelDocHdrId" column="CNCL_DOC_HDR_ID" jdbc-type="VARCHAR" />
     	<field-descriptor name="versionNumber" locking="true" column="VER_NBR" jdbc-type="BIGINT" />
+    	<field-descriptor name="active" column="ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion" />
```
Comment (nkimble) [inline src/main/resources/com/rsmart/kuali/kfs/cr/ojb-cr.xml:36]: Are these changes part of KFSPTS-30132? I don't think this file should be in this PR.

**Review by nkimble — CHANGES_REQUESTED** [review]
I think some of the files for a different user story are included in this PR. Could you please take a look.

**Comment by ajd299** [comment]
Thanks for catching that I have reversed that commit and it there should no longer be any CheckReconciliation code in this Pull Request

**Comment by nkimble** [comment]
These code changes look good. 
I did have a question regarding the nonprod-sql changes previously merged on PR 1334 in that repository.

Asking the question: Will we now need (should we have) an index on this new data value since it has been added to the Document Search?

**Comment by ajd299** [comment]
Please see https://github.com/CU-CommunityApps/nonprod-sql/pull/1340

**Review by nkimble — APPROVED** [review]
My concerns have been address and related/required SQL changes are also ready. Approving for merge.

**Pull request merged**